.travis.yml

```diff
@@ -8,26 +8,33 @@ php:
   - 7.0
   - hhvm
 
-matrix:
-  allow_failures:
-    - php: 7.0
-    - php: hhvm
+sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
+
+matrix:
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS="--prefer-lowest"
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
 
 before_script:
-  - composer self-update
-  - composer install --prefer-source --no-interaction
+  - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - composer selfupdate
+  - composer config -q github-oauth.github.com $GITHUB_OAUTH_TOKEN
+  - travis_wait composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
 
 script:
-  - mkdir -p build/logs
-  - mkdir -p build/cov
-  - php bin/phpunit -d error_reporting=16384
```

Comment:
16384  = User-generated warning message. This is like an E_DEPRECATED, except it is generated in PHP code by using the PHP function trigger_error(). See (php doc about errorfunc constants](http://php.net/manual/en/errorfunc.constants.php). 
This error-reporting conf was introduced to fix #45.